TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/LiftRedux.java

```diff
@@ -0,0 +1,189 @@
+package org.firstinspires.ftc.teamcode.subsystems;
```

Comment:
Why LiftRedux and also Lift? It looks to me like LiftRedux is just the old code from Lift.